core/src/main/java/com/predic8/membrane/core/lang/AbstractScriptInterceptor.java

```diff
@@ -69,7 +70,15 @@ protected Outcome runScript(Exchange exc, Flow flow) throws InterruptedException
 
         Message msg = getMessage(exc, flow);
 
-        Object res = script.apply(getParameterBindings(exc, flow, msg));
+        Object res;
+        try {
+            res = script.apply(getParameterBindings(exc, flow, msg));
+        } catch (Exception e) {
+            Map<String,Object> details = new HashMap<>();
+            details.put("message","See logs for details.");
+            exc.setResponse(createProblemDetails(500, "/internal-error", "Internal Server Error", details));
```

Comment:
shouldn't this be something like `createProblemDetails(500, "/scripting-error"` ?

Reply:
I also thought about that. But do we want to expose internals to the outside, like the fact that there is a script?

Reply:
So it's related to #476 ...